app/views/static/buttons/generic_button_component.html.haml

```diff
@@ -0,0 +1,27 @@
+.clearfix
```

Comment:
I missed your above comment before.

A lot has happened since that comment :)
Anyway, to recap, 
- `generic-button-component` is now called `form-button-component` and is the helper of the main form component.
- As we convert other forms to components, we can re-use `form-button-component` and cut down all the repeat code.